scripts/publish-schematics-verdaccio.sh

```diff
@@ -1,91 +0,0 @@
-###########################################################
```

Comment:
😢

Reply:
Life's brutal, but served us well for quite some time.

F